docs/tritonfrontend/README.md

```diff
@@ -0,0 +1,115 @@
+### Triton Server (tritonfrontend) Bindings
```

Comment:
Is this a duplicate of `server/python/README.md`? If so, let's remove this one.

Reply:
I see Guan asked you to move the other one here: https://github.com/triton-inference-server/server/pull/7501/files#r1725941137 - so you can remove the other instead and keep the `docs/` one.

Reply:
I removed the `server/python/README.md`. I believe the final version was requested to be within `docs`.